api/proto/storage/sql.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+package proto.storage;
+
+option go_package = "github.com/spinnaker/kleat/api/client/storage";
+
+import "google/protobuf/wrappers.proto";
+
+message SQL {
+  // Whether this persistent store is enabled.
+  google.protobuf.BoolValue enabled = 1;
+
+  // Default database connection pool.
+  ConnectionPools connectionPools = 2;
+
+  // Migration connection pool
+  ConnectionPool migration = 3;
+
+  // The default connection pool
+  message ConnectionPools {
+      ConnectionPool default = 1;
+  }
+
+  // ConnectionPool confifugration for the SQL server
+  message ConnectionPool {
+      // Database username
+      string user = 1;
+      // Database password
+      string password = 2;
+      // Database connection string. This needs to include server port
+      // and database name as well
+      string jdbcUrl = 3;
+      // Database connection timeout
+      int32 connectionTimeout = 4;
+      // maxLifetime controls the maximum lifetime of a connection in the pool.
+      int32 maxLifetime = 5;
```

Comment:
Is the unit for `connectionTimeout` and `maxLifetime` seconds? Might be worth clarifying.

Reply:
they are in milliseconds. updated the comments.

Reply:
👍  Thanks!